examples/example_cori_haswell.cfg

```diff
@@ -0,0 +1,128 @@
+# Do not edit any example cfg except for `example_generic.cfg`!
+
+[default]
+case = 20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis
+environment_commands = "source /global/common/software/e3sm/anaconda_envs/load_latest_e3sm_unified_cori-haswell.sh"
+input = /global/cscratch1/sd/forsyth/zppy_complete_run_nersc_output/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis
+input_subdir = archive/atm/hist
+mapping_file = /global/homes/z/zender/data/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc
+output = /global/cscratch1/sd/forsyth/zppy_complete_run_nersc_output/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis
+partition = haswell
+walltime = "02:00:00"
+www = /global/cfs/cdirs/e3sm/www/forsyth/zppy_complete_run_nersc_output
+
+[climo]
+active = True
+years = "51:55:2", "51:55:4",
+
+  [[ atm_monthly_180x360_aave ]]
+  frequency = "monthly"
+
+  [[ atm_monthly_diurnal_8xdaily_180x360_aave ]]
+  frequency = "diurnal_8xdaily"
+  input_files = "eam.h4"
+  vars = "PRECT"
+
+[ts]
+active = True
+frequency = "monthly"
+years = "51:55:2",
+
+  [[ atm_monthly_180x360_aave ]]
+  input_files = "eam.h0"
+
+  [[ atm_daily_180x360_aave ]]
+  frequency = "daily"
+  input_files = "eam.h1"
+  vars = "PRECT"
+
+  [[ atm_monthly_glb ]]
+  input_files = "eam.h0"
+  input_subdir = "archive/atm/hist"
+  mapping_file = "glb"
+  years = "51:61:5",
+
+  [[ land_monthly ]]
+  input_files = "elm.h0"
+  input_subdir = "archive/lnd/hist"
+  vars = "FSH,LAISHA,LAISUN,RH2M"
+
+  [[ rof_monthly ]]
+  extra_vars = 'areatotal2'
+  input_files = "mosart.h0"
+  input_subdir = "archive/rof/hist"
+  mapping_file = ""
+  vars = "RIVER_DISCHARGE_OVER_LAND_LIQ"
+
+[tc_analysis]
```

Comment:
@chengzhuzhang It's possible #169 (which added Tropical Cyclone analysis/diagnostics) wasn't tested on Cori. This task worked fine on the Compy example (#234). The Cori example `cfg`, however, fails with the following. Any thoughts?

```
Fatal error in MPI_Init: Other MPI error, error stack:
MPIR_Init_thread(537):
MPID_Init(246).......: channel initialization failed
MPID_Init(647).......:  PMI2 init failed: 1
/var/spool/slurmd/job58523789/slurm_script: line 94: 42021 Aborted                 DetectNodes --verbosity 0 --in_connect ${result_dir}connect_CSne${res}_v2.dat --closedcontourcmd "PSL,300.0,4.0,0;_AVG(T200,T500),-0.6,4,0.30" --mergedist 6.0 --searchbymin PSL --outputcmd "PSL,min,0;_VECMAG(UBOT,VBOT),max,2" --timestride 1 --in_data_list ${result_dir}inputfile_${file_name}.txt --out ${result_dir}out.dat
```

Reply:
This looks like the kind of error that happens when you're using the wrong MPI (like from conda-forge).  I might be on completely the wrong track but wanted to suggest that in case...

Reply:
I recall that there was an issue when you tested the tc_analysis.bash on Cori. Not sure if this is related. but from what I can tell, on Cori compute node, it tries to initiate an MPI process with the tempestextremes call (DetectNodes), maybe you can try following? , i.g.:
```
"srun -n 32 DetectNodes --verbosity 0 --in_connect ${result_dir}connect_CSne${res}_v2.dat --closedcontourcmd "PSL,300.0,4.0,0;_AVG(T200,T500),-0.6,4,0.30" --mergedist 6.0 --searchbymin PSL --outputcmd "PSL,min,0;_VECMAG(UBOT,VBOT),max,2" --timestride 1 --in_data_list ${result_dir}inputfile_${file_name}.txt --out ${result_dir}out.dat
```

Reply:
hmmm, I would suggest to get the stand alone DetecNodes command on has-well, with and without `srun` and see if the error would reproduce.

Reply:
Can you also do `which DetectNodes` to make sure it's coming from Spack and not from conda-forge?